app/controllers/admin/neurons_controller.rb

```diff
@@ -74,11 +74,16 @@ def delete
 
     def restore
       neuron.paper_trail_event = "restore"
-      neuron.update! deleted: false
+      neuron.update! state: "pending"
```

Comment:
I'd say it's okay as they were deleted for a reason anyway. But it just might not pass the _QA tests_